### ISSUE TITLE
Add database reading functions

### ISSUE THREAD

**Issue by BryceFury** [BryceFury/tumbly#9]
Read from the database back into the same variables as were written for future use.

**Comment by xssChauhan** [comment]
I think i can help you out.
Let me get started with the code.
Any headup?

**Comment by BryceFury** [comment]
Nope, just stick to PEP8 for style unless absolutely necessary otherwise, looking forward to your PR!
